publisher/internal/analytics/ethereum/operations.go

```diff
@@ -448,16 +454,23 @@ func (p Position) CanPublish() bool {
 		log.Printf("SKIP - missing price (could not calculate current ratio). Tx: %s\n\n", p.TxHash)
 		return false
 	}
-
+	if !isNativeInvolved(p) && !isStableInvolved(p) {
+		log.Printf("SKIP - no stable or native currency involved. Tx: %s\n\n", p.TxHash)
+		return false
+	}
 	return true
 }
 
 func (p Position) areTokensSet() bool {
 	return (!strings.EqualFold(p.Token0.Address, "") && !strings.EqualFold(p.Token1.Address, ""))
 }
 
-func (p Position) isToken0StableAndToken1Native() bool {
-	return (strings.EqualFold(p.Token1.Address, addressWETH) || strings.EqualFold(p.Token0.Address, addressUSDC) || strings.EqualFold(p.Token0.Address, addressUSDT))
+func (p Position) isToken1Stable() bool {
```

Comment:
Drawing on the previos idea about `isStableInvolved` perhaps this and the second function below may be refactored in a similar way? e.g. you might ask if Token1 is stable by calling this refactored method as `isToken1OneOf(stabeCoins...)`